activerecord/lib/active_record/relation/delegation.rb

```diff
@@ -89,6 +89,8 @@ def method_missing(method, *args, &block)
             self.class.delegate_to_scoped_klass(method)
             scoping { @klass.public_send(method, *args, &block) }
           elsif arel.respond_to?(method)
+            ActiveSupport::Deprecation.warn \
+              "Delegating #{method} to arel is deprecated and will be removed in Rails 6.0."
```

Comment:
We should probably lose the `delegate` call below, so the deprecation message appears for every call.